### PR TITLE
Added important note #5453

### PR DESCRIPTION
**Pull request by j0rt3g4**
Added Important Note #5453

### CODE REVIEW

**Comment by j0rt3g4** [comment]
Copy review @nenonix @JohanFreelancer9 is required on this one guys. Thank you in advance!

File: windows/security/threat-protection/windows-defender-system-guard/system-guard-secure-launch-and-smm-protection.md [hunk 1]
```diff
@@ -54,6 +54,9 @@ Click **Start** > **Settings** > **Update & Security** > **Windows Security** >
 
 ![Secure Launch Registry](images/secure-launch-registry.png)
 
+> [!IMPORTANT]
+> If the system guard enabled as a registry key, then standard hardware security is not available for Intel i5 7200U processor".
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-system-guard/system-guard-secure-launch-and-smm-protection.md]:
```suggestion
> If the system guard is enabled as a registry key, then standard hardware security is not available for the Intel i5 7200U processor.
```

**Comment by mypil** [comment]
@Dansimp - PR has been copy edited and is ready for final review. This is based on issue #5453

Thank you.

cc: @kenwith

**Comment by illfated** [comment]
@JohanFreelancer9 : Just a small afterthought about the name "System Guard" - the added Note uses a lowercase service name `system guard` instead of capitalized System Guard, in contrast to the rest of the page. Any thoughts, is it purely cosmetic or is it considered a proper name for the service?

**Comment by Dansimp** [comment]
Quick copy edit. @illfated is correct, System Guard is a feature name and should be capitalized.

**Comment by j0rt3g4** [comment]
@Dansimp  so should I create another PR?

**Comment by illfated** [comment]
@j0rt3g4 : No need, the requested change was implemented in commit https://github.com/MicrosoftDocs/windows-itpro-docs/pull/5571/commits/1a3163d419aa3033907213155a881d62b7788b24 , right before the Merge commit (https://github.com/MicrosoftDocs/windows-itpro-docs/commit/dc5ef67ba54687655f973d9c7aa490fca8f772c0).

**Comment by j0rt3g4** [comment]
@illfated  great! thank you

**Pull request merged**